### PR TITLE
Debugkit docs

### PR DESCRIPTION
**Pull request by markstory**
Remove debugkit documentation now that it is available through the plugin-docs site.

### CODE REVIEW

File: fr/contents.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ Contenu
     :caption: Plugins
 
     chronos
-    debug-kit
+    debug-kit </debugkit>
```
Comment (garas) [inline fr/contents.rst]:
```suggestion
    debug-kit </debugkit/3.x/fr/>
```

File: fr/debug-kit.rst [hunk 2]
```diff
@@ -1,233 +1,4 @@
...
-Pour utiliser un panneau de plugin ou de l'application, mettez à jour
-la configuration du DebugKit de votre application pour ajouter le panneau::
-
-    // dans config/bootstrap.php
-    Configure::write('DebugKit.panels', ['App', 'MyPlugin.MyCustom']);
-    Plugin::load('DebugKit', ['bootstrap' => true]);
-
-Ce qui est au-dessus charge tous les panneaux par défaut ainsi que le panneau
-``AppPanel``et le panneau ``MyCustomPanel`` depuis ``MyPlugin``.
+Cette page a été `déplacée </debugkit/fr/>__.
```
Comment (garas) [inline fr/debug-kit.rst]:
```suggestion
Cette page a été `déplacée </debugkit/3.x/fr/>__.
```

File: ja/contents.rst [hunk 3]
```diff
@@ -74,10 +74,10 @@
     :maxdepth: 3
     :caption: プラグイン
 
-    authorization <https://book.cakephp.org/authorization/>
-    authentication <https://book.cakephp.org/authentication/>
+    authorization </authorization/>
+    authentication </authentication/>
     chronos
-    debug-kit
+    debug-kit </debugkit/>
```
Comment (garas) [inline ja/contents.rst]:
```suggestion
    debug-kit </debugkit/3.x/jp/>
```

Should it be `ja`?

As I just learned, `jp` is country code, `ja` is language code.

File: ja/debug-kit.rst [hunk 4]
```diff
@@ -1,267 +1,4 @@
...
-
-上記は、すべてのデフォルトのパネルと同じように ``AppPanel`` と ``MyPlugin`` の
-``MyCustomPanel`` パネルを読みこみます。
-
-ヘルパー関数
-================
-
-* `sql()` ORM クエリーから SQL をダンプします。
-* `sqld()` ORM クエリーから SQL をダンプして終了します。
+このページは移動 `しました</debugkit/jp/>`__.
```
Comment (garas) [inline ja/debug-kit.rst]:
```suggestion
このページは移動 `しました</debugkit/3.x/jp/>`__.
```

File: pt/contents.rst [hunk 5]
```diff
@@ -75,7 +75,7 @@ Conteúdo
     :caption: Plugins
 
     chronos
-    debug-kit
+    debug-kit </debugkit/>
```
Comment (garas) [inline pt/contents.rst]:
```suggestion
    debug-kit </debugkit/3.x/pt/>
```

File: pt/debug-kit.rst [hunk 6]
```diff
@@ -1,215 +1,4 @@
...
-de sua aplicação para incluir o painel::
-
-    Configure::write(
-        'DebugKit.panels',
-        array_merge(Configure::read('DebugKit.panels'), ['MyCustomPanel'])
-    );
-
-O código acima deve carregar todos os painéis padrão tanto como os outros
-painéis customizados do ``MyPlugin``.
+Esta página foi `movida </debugkit/pt/>`__.
```
Comment (garas) [inline pt/debug-kit.rst]:
```suggestion
Esta página foi `movida </debugkit/3.x/pt/>`__.
```

File: en/debug-kit.rst [hunk 7]
```diff
@@ -1,284 +1,4 @@
...
-
-The above would load all the default panels as well as the ``AppPanel``, and
-``MyCustomPanel`` panel from ``MyPlugin``.
-
-Helper Functions
-================
-
-* `sql()` Dumps out the SQL from an ORM query.
-* `sqld()` Dumps out the SQL from an ORM query, and exits.
+This page has `moved </debugkit/en/>`__.
```
Comment (garas) [inline en/debug-kit.rst]:
```suggestion
This page has `moved </debugkit/3.x/en/>`__.
```

File: ja/contents.rst (hunk 3 above)
Comment (markstory) [reply ja/contents.rst]: It should be `ja` I'll fix the debug kit code.

**Pull request merged**